Ascend.java

```diff
@@ -2,6 +2,27 @@
 // and prints them in increasing order.

```

Comment:
OVERALL:
1. This is a good submission
2. My only main issue is styling. please go over the styling guidelines

Reply:
אני סקפטי שיש בעיה של הזחות בgithub, במיוחד שיש שפות תכנות שנשענות על הזחות. 
בדגש על TimeFormat..

Reply:
לגבי Bill3, לא מפצלים הכרזה והצבת ערך ראשוני. זה bad practice
נניח משנים טיפה את הקוד. והוא כבר לא יעבוד. דוגמה נמצאת למטה.
יש עוד דוגמאות בנושא.. 
לכן, ההעדפה היא לעשות את זה באותה שורה.. 

```
double priceForEach;

priceForEach += Math.ceil((double)totalPrice/3);
```